app/assets/javascripts/components/project_editor.js.jsx.coffee

```diff
@@ -1,14 +0,0 @@
-div = React.createFactory 'div'
-img = React.createFactory 'img'
-@ProjectEditor = React.createClass
-  handleRemove: (e) ->
-    $(@refs["row"]).fadeOut( => @.props.onRemoveItem() )
-
-  render: ->
-    div className: "row editors-edit-row fade in", ref: "row",
-      div className: "col-md-3",
-        img src: this.props.editor.avatar, className:"activitar"
-      div className:"col-md-9",
-        "#{@props.editor.name} <#{@props.editor.email}>"
-      div className:"remove", onClick:@.handleRemove,
-        "×"
```

Comment:
despite all the atrocity, this isn't a terrible template to work with. jsx can so claustrophobic.